SuccessEycaActivationActivity/__tests__/handler.test.ts

```diff
@@ -0,0 +1,114 @@
+/* tslint:disable: no-any */
+import { addYears } from "date-fns";
+import { left, right } from "fp-ts/lib/Either";
+import { none, some } from "fp-ts/lib/Option";
+import { fromLeft, taskEither } from "fp-ts/lib/TaskEither";
+import { toCosmosErrorResponse } from "io-functions-commons/dist/src/utils/cosmosdb_model";
+import { FiscalCode, NonEmptyString } from "italia-ts-commons/lib/strings";
+import { context } from "../../__mocks__/durable-functions";
+import * as cgn_checks from "../../utils/cgn_checks";
+import * as codeUtils from "../../utils/cgnCode";
+import {
+  ActivityInput,
+  getSuccessEycaActivationActivityHandler
+} from "../handler";
+
+const aFiscalCode = "RODFDS92S10H501T" as FiscalCode;
+const aCardCode = "AAAABBBBCCCCDDDD" as NonEmptyString;
+
+const findLastVersionByModelIdMock = jest
+  .fn()
+  .mockImplementation(() => taskEither.of(some({})));
+const updateMock = jest.fn().mockImplementation(() => taskEither.of({}));
+
+const userEycaCardModelMock = {
+  findLastVersionByModelId: findLastVersionByModelIdMock,
+  update: updateMock
+};
+
+const anActivityInput: ActivityInput = {
+  fiscalCode: aFiscalCode
+};
+const extractEycaExpirationDateMock = jest
+  .spyOn(cgn_checks, "extractEycaExpirationDate")
+  .mockImplementation(() => right(addYears(new Date(), 5)));
+
+const genRandomCardCodeMock = jest
+  .spyOn(codeUtils, "genRandomCardCode")
+  .mockImplementation(() => Promise.resolve(aCardCode));
+describe("UpdateCgnStatusActivity", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it("should return success if card activation succeded", async () => {
+    const handler = getSuccessEycaActivationActivityHandler(
+      userEycaCardModelMock as any
+    );
+    const response = await handler(context, anActivityInput);
+    expect(response.kind).toBe("SUCCESS");
```

Comment:
Can we add an expect on `userEycaCardModel.update` method to verify the right function execution?